eng/Version.Details.xml

```diff
@@ -37,6 +37,10 @@
       <Uri>https://github.com/dotnet/aspnetcore</Uri>
       <Sha>9abbf4e1134cf1e699879756d1370c1d7c52e701</Sha>
     </Dependency>
+    <Dependency Name="System.Composition" Version="1.0.31.0">
```

Comment:
Should remove this. And change the System.Composition package in the Version.props to be in the "Manual" section. I just noticed you added it t o the automated section